Catalog doc (#95)
miamtech/mealz-documentation: docs/android/features/catalog/how-to-use.mdx

---
sidebar_position: 2
label: "How to use"
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

# How to use Catalog 

> As we say in previous page there si two way to use catalog :
> 
>     - Basic implementation : an all in one compoment that embed naviagation and all needed other component
>     - Custom implementation : were you can handle naviagtion by yourself and link component that you need
> 
> ## Basic implementation 
> 
> For this implementation you will need CatalogJourney component, it's an all in one component mergin all available feature of catalog.
> including direct link to recipe collection. 

Reviewer: ```suggestion
As we said on the previous page, there are two ways to use the catalog :

    - Basic implementation : an all-in-one component that embeds the navigation and all needed other components
    - Custom implementation : were you can handle navigation by yourself and link components that you need

## Basic implementation 

For this implementation you will need CatalogJourney component, it's an all-in-one component merging all the available features of the catalog.
It includes a direct link to recipe collections. 
```

Author: ok
